src-built-in/components/myApps/src/index.jsx

```diff
@@ -93,12 +93,20 @@ class AppLauncher extends React.Component {
 }
 
 fin.desktop.main(function () {
-	FSBL.addEventListener("onReady", function () {
-		createStore((store) => {
-			storeActions.initialize(() => {
-				ReactDOM.render(<AppLauncher />,
-					document.getElementById("wrapper"));
+	if (window.FSBL && FSBL.addEventListener) { 
+		FSBL.addEventListener("onReady", FSBLReady); 
+	} else {
+		window.addEventListener("FSBLReady", FSBLReady) 
+	} 
+
+	function FSBLReady(){
+		FSBL.addEventListener("onReady", function () {
```

Comment:
This appears to be listening for the event inside the event handler. I think you need to remove this line.

Reply:
my mistake @NinjaManatee will update